packages/app/src/crash-dump/actions/send-crash-dump-data.action.test.ts

```diff
@@ -17,6 +17,9 @@ import createFreshdeskTicket from "Renderer/utils/create-freshdesk-ticket/create
 
 const crashDumpsMock: string[] = ["/pure/logs/crash-dumps/file.hex"]
 
+const MuditaOSLogs = new File([""], "MuditaOS.log", { type: "text/html" })
+const logsFiles: File[] = [MuditaOSLogs]
```

Comment:
```suggestion
const muditaOSLogs = new File([""], "MuditaOS.log", { type: "text/html" })
const logsFiles: File[] = [muditaOSLogs]
```

Reply:
name changed in [9f185d4](https://github.com/mudita/mudita-center/pull/586/commits/9f185d4bd7b2891cecf5832c9dbe6ecf43e73203)